### DISCUSSION TITLE
AOSP 11.0 v305

### DISCUSSION THREAD

**Discussion by phhusson** [phhusson/treble_experimentations#1804] [General]
- April security patch level (for real)
- Fix in-call echo in some Mediatek devices ( Thanks @PeterCxy )

<hr /><em>This discussion was created from the release <a href='https://github.com/phhusson/treble_experimentations/releases/tag/v305'>AOSP 11.0 v305</a>.</em>

**Comment by mschabhuettl** [comment]
Why "for real"? Just curious?

**Reply by 00p513-dev** [reply]
https://github.com/phhusson/treble_manifest/commit/75ff417afaa62891fcf34501ca618bca7e848ce3

**Comment by Gyulank** [comment]
Device is not Play Protect certified. I tried back from 305, so 304, and 303 also have this problem. And Always opening the menu ín Chrome without any reason. 11.0 v305 ab gapps Doogee S40

**Reply by 00p513-dev** [reply]
See the uncertified note in @phhtreble on telegram with regards to your Play Protect issues. Are you using the GAPPS build?

**Reply by Gyulank** [reply]
Of course. The same build tried with different versions. Now the 302 have, and it have not this problem. Because of it I wrote that 305, 304, 303 have.

**Reply by 00p513-dev** [reply]
see uncertified note

**Reply by Gyulank** [reply]
I sad this problem only from 303 build. What need i see? I know that from 303 don't work and don't my fault.

**Reply by 00p513-dev** [reply]
@phhtreble on telegram

**Reply by Gyulank** [reply]
If you know what do you want to see or what need to i know, say that! I wrote a problem with these builds. I don't care anything else. Before 303 work, after don't work. I want that in 306 will work.

**Reply by Osanosa** [reply]
No one has to do anything for you, remember, you're not paying for it, so shut the fuck up and join Telegram group or read FAQ to get answer to every possible question.

And since you're **that** lazy ass I'll make a quote for ya:
Register your device on https://www.google.com/android/uncertified/

To get the ID, you can either:
- adb root, then adb shell, press enter, and at the new prompt, inside adb shell, do sqlite3 /data/data/com.google.android.gsf/databases/gservices.db    'select * from main where name = "android_id";'
- https://play.google.com/store/apps/details?id=com.evozi.deviceid => Google Service Framework (GSF) ID
- for multiuser see https://github.com/phhusson/treble_experimentations/issues/1794#issuecomment-816632811

**Reply by Gyulank** [reply]
But until these builds worked... Now why don't? And i haven't current, so i haven't pc, so i haven't connect, so i don't know my ssaid.

**Reply by Osanosa** [reply]
Oh God, just install damn [app](https://play.google.com/store/apps/details?id=com.evozi.deviceid), copy GSF ID and register your device on a website

**Reply by Gyulank** [reply]
But that register site sad that need SSAID, not GFSID.

**Reply by Osanosa** [reply]
Just do what instruction says

**Reply by Gyulank** [reply]
No! When I tap in the text box where need write the SSAID, give me a numerical keyboard, but in the GSF not just numbers have.

**Reply by Osanosa** [reply]
Convert hex to decimal

**Comment by Michael8878** [comment]
this version does not work for the Samsung Galaxy A70 (A705MN), it gives infinite loop. appears 1 second to configure the android, and returns to the infinite loop again. Can someone help me?

**Reply by 00p513-dev** [reply]
join @phhtreble and gib logs
